decidim-budgets/app/views/decidim/budgets/admin/projects/index.html.erb

```diff
@@ -1,16 +1,16 @@
 <% add_decidim_page_title(t(".title")) %>
 <div class="card">
-  <div class="card-divider pb-4 border-b border-gray mb-4">
-    <h2 class="text-xl font-bold inline-block">
+  <div class="item_show__header">
+    <h2 class="item_show__header-title">
       <div>
         <%= link_to translated_attribute(budget.title), budgets_path %> &gt;
         <%= t(".title") %>
         <span id="js-selected-resources-count" class="component-counter component-counter--inline" title="<%= t("decidim.budgets.admin.projects.index.selected") %>"></span>
       </div>
+      <%= render partial: "bulk-actions" %>
```

Comment:
This makes the button list inside to look bad, with a text bold that's inherited from the title class:

![image](https://github.com/decidim/decidim/assets/717367/e098a941-a4e3-4a2e-82bb-0866a95e51a3)


Also mind that the bulk actions buttons are broken. I had to correct them with this change:

```diff
diff --git a/decidim-budgets/app/views/decidim/budgets/admin/projects/bulk_actions/_dropdown.html.erb b/decidim-budgets/app/views/decidim/budgets/admin/projects/bulk_actions/_dropdown.html.erb
index 07a75cd8d7..eef8b2a990 100644
--- a/decidim-budgets/app/views/decidim/budgets/admin/projects/bulk_actions/_dropdown.html.erb
+++ b/decidim-budgets/app/views/decidim/budgets/admin/projects/bulk_actions/_dropdown.html.erb
@@ -1,7 +1,7 @@
 <div id="js-bulk-actions-wrapper">
   <button
     id="js-bulk-actions-button"
-    class="button button__sm button__secondary button--nomargin tiny success light dropdown hide"
+    class="button button__sm button__transparent button__transparent-secondary dropdown
     type="button"
     data-toggle="js-bulk-actions-dropdown">
     <%= t("decidim.budgets.admin.projects.index.actions") %>
``` 

We also need to fix the bulk action button in proposals https://github.com/decidim/decidim/blob/28d00c3c0cc656cef32f2891d6d395282c686cbc/decidim-proposals/app/views/decidim/proposals/admin/proposals/bulk_actions/_dropdown.html.erb#L4

Reply:
![image](https://github.com/decidim/decidim/assets/105683/75c20e2e-bb8e-4aa0-b8dc-bb6d6140f191)

Fixed in 6d38b502f9